tests/FSharp.Compiler.Private.Scripting.UnitTests/DependencyManagerInteractiveTests.fs

```diff
@@ -719,6 +719,7 @@ x |> Seq.iter(fun r ->
             """    #time ["on"|"off"];;                          // Toggle timing on/off"""
             """    #clear;;                                      // Clear screen"""
             """    #help;;                                       // Display help"""
+            """    #help "expr";;                                // Display documentation for an expression, e.g. #help "List.map";;"""
```

Comment:
I dislike bike shedding and words are super hard.  But (gonna do it anyway :-) ) this isn't really an expression.  I think either symbol, member or identifier may be a better word.  I think my preference is identifier, but am open to better suggestions.

Reply:
Good point, I changed it to "identifier".